### DISCUSSION TITLE
My product branding as default CSS

### DISCUSSION THREAD

**Discussion by ruidiasbraga** [zauberzeug/nicegui#2645] [Q&A]
### Question

Hi, 

How can I change the default CSS in such a way that when I do ui.button('Label') the result is my personal branding and not the default nicegui css. 

I saw options to add more css on top, but that will make the code longer, and I need to make use of properties like "!important" to make my changes take effect in the screen. I would like to change some colors and shapes of nicegui buttons, dropdowns etc.. 

I would like to:
-  ui.button('Label')

 
 and not 
- ui.button('Label').classes('my-btn-primary my-btn-clicackle')

**Comment by ghost** [comment]
Hi  @ruidiasbraga ,

How do you like a solution like this. I use it myself in my project.

```py
brand_color = '#56078f'


@ui.page('/index')
def index() -> None:
    ui.colors(primary=brand_color)
```

By the way I moved my brand related settings to a separate config.py file.

As I know the last line of code have to be added to each @ui.page function cause the are not shared.

**Comment by falkoschindler** [comment]
You can also use [default classes](https://nicegui.io/documentation/element#default_classes) for UI elements:

```py
ui.button.default_classes('flat bg-orange-8')

...

ui.button('Click me')
```
